go.sum

```diff
@@ -88,11 +224,15 @@ gopkg.in/Clever/kayvee-go.v6 v6.27.0/go.mod h1:G0m6nBZj7Kdz+w2hiIaawmhXl5zp7E/K0
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
```

Comment:
still see the kayvee-go v6 dependency here--is that legit? Or maybe a `go mod tidy` will remove it?

Reply:
It's still indirect required because the batch consumer requires kayvee.v6. Will this cause a dependancy issue?

Reply:
it shouldn't be an issue! The otel dependency issue that plagues us arises when you indirectly pull in conflicting versions of the opentelemetry collector. Otel was introduced with kayvee v7 so you're good. You'd know you're not good if it failed to compile ha